src/plugins/timelist/Timelist.vue

```diff
@@ -197,23 +197,28 @@ export default {
                 this.listActivities();
             }
         },
-        updateTimestamp(_bounds, isTick) {
+        updateTimestamp(bounds, isTick) {
             if (isTick === true && this.openmct.time.clock() !== undefined) {
                 this.updateTimeStampAndListActivities(this.openmct.time.clock().currentValue());
+            } else if (isTick === false && this.openmct.time.clock() === undefined) {
+                // set the start time for fixed time using the selected bounds start
+                this.updateTimeStampAndListActivities(bounds.start);
             }
         },
         setViewFromClock(newClock) {
             this.filterValue = this.domainObject.configuration.filter;
-            const isFixedTime = newClock === undefined;
-            if (isFixedTime) {
+            this.isRealTime = newClock !== undefined;
+            if (!this.isRealTime) {
                 this.hideAll = false;
                 this.showAll = true;
                 this.updateTimeStampAndListActivities(this.openmct.time.bounds()?.start);
             } else {
-                this.setSort();
-                this.setViewBounds();
                 this.updateTimeStampAndListActivities(this.openmct.time.clock().currentValue());
             }
+
+            this.setSort();
+            this.setViewBounds();
+            this.listActivities();
```

Comment:
The `this.updateTimeStampAndListActivities` method already performs `this.listActivities` so this will result in it being run twice. Can this be done differently?

Reply:
Absolutely, I think this was an artifact of the PR being based on an earlier version of master that predated the `updateTimestampAndListActivities` method and then reintroduced from a merge conflict. I've removed it as it did indeed duplicate the 3 method calls.